src/js/tabs/pid_tuning.js

```diff
@@ -1715,13 +1716,26 @@ pid_tuning.initialize = function (callback) {
                 };
             }
 
+            function calculateThrottlePosition(thrPercent, canvasWidth, canvasHeight, mid, midx, midxl, midxr, midy, midyr, midyl) {
```

Comment:
Suggestion that the arguments `midxl`, `midxr`, `midyl` and `midyr` are declared in same order, eg:

```suggestion
            function calculateThrottlePosition(thrPercent, canvasWidth, canvasHeight, mid, midx, midxl, midxr, midy, midyl, midyr) {
```

Reply:
You are right, I will change it.

Reply:
Fixed!